frontend/src/pages/modelCatalog/ModelCatalogPage.tsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import { ModelCatalogContext } from '~/concepts/modelCatalog/context/ModelCatalogContext';
+
+const ModelCatalogPage: React.FC = () => {
```

Comment:
This page actually isn't rendered by any route - I think the page component was already created under `frontend/src/pages/modelCatalog/screens/ModelCatalog.tsx`, you can delete this file and use that one to test taking your data from context instead